deploy/scripts/src/accounts/test.ts

```diff
@@ -128,6 +167,28 @@ export function setup (): void {
   describeProfile(loadProfile)
 }
 
+type mfaType = 'SMS' | 'AUTH_APP'
+interface validateUserData {
+  email: string
+  mfaOption: mfaType
+}
+const validateData: validateUserData[] = new SharedArray('data', () => Array.from({ length: 10000 },
+  (_, i) => {
+    const id: string = Math.floor((i / 2) + 1).toString().padStart(5, '0')
+    if (i % 2 === 0) {
+      return {
+        email: `perftestam1_app_${id}@digital.cabinet-office.gov.uk`,
+        mfaOption: 'AUTH_APP' as mfaType
+      }
+    } else {
+      return {
+        email: `perftestam1_sms_${id}@digital.cabinet-office.gov.uk`,
+        mfaOption: 'SMS' as mfaType
+      }
+    }
+  }
+))
+
```

Comment:
This data no longer needs to be fixed in a shared array as the stubs accept any email. It can be done in a similar way to the `changeEmail` scenario